### ISSUE TITLE
Example ic_engine.py heat release rate far too high

### ISSUE THREAD

**Issue by boroli** [Cantera/cantera#819]
**System information**

- Cantera version: 2.5 (current dev version, pulled from git yesterday)
- OS: debian10 
- Python/MATLAB version: python3.7

**Actual behavior**

I was running the ic_engine.py and the output of
'Heat release rate per cylinder (estimate):'
is far too high, something like 1.3 GW!
That's, considering the amount of fuel injected, impossible.
The work is less than 20kW and seems to be reasonable.

The time resolved heat release is computed during simulation as:
```python
heat_release_rate = - cyl.volume * ct.gas_constant * cyl.T * \
        np.sum(gas.standard_enthalpies_RT * cyl.thermo.net_production_rates, 0)
```
It's always positive, whereas the second line is always negative, probably the walls are adiabatic, and hence there's no quenching in the model for that. I assume that should not be of a big issue.
But I didn't understand completely what's done here, especially not in the second line (what's the zero in the end of the line, supposed to do?). I also checked the units and it should be result in J/s, and be correct. But the heat release during combustion should be the integral c_v dT from beginning to end for constant volume combustion (I know that this is not the ideal case here). But this seems to be only the time revolved chemical energy, but the actual integration is done over time:
```python
# heat release
Q = trapz(states.heat_release_rate, t)
print('{:45s}{:>4.1f} kW'.format('Heat release rate per cylinder (estimate):',
                                 Q / t[-1] / 1000.))
```
I didn't check the math here, if you can easily exchange coordinates from integration over temperature with over time. Anyway the result is far too high, and it comes mainly from the sum over the production rates.

**Additional context**

I also computed the AFR, just to check if something is wrong in general (because the fuel was changed from propane to C12H26 in the example a few time ago, but the fuel amount remained), but it's about 16. So seemed to be ok:
```python
# AFR
airIn = trapz(states.mdot_in,t)
AFR = airIn/(injector_mass*sim_n_revolutions*0.5)
print("AFR",AFR)
```
Please let me know, if you want to add these lines to the example, and if you need a merge request or something like that, for it.

**Comment by ischoegl** [comment]
Interesting. I updated this example recently, but didn’t look into specifics about some of the pre-existing metrics. I had mostly looked at efficiencies, which I recall being reasonable (considering that walls are modeled adiabatic).

PS: I just verified this behavior on my installation. It is quite odd as I recall the efficiency being ok previously (it is no longer ok).

**Comment by ischoegl** [comment]
OK. The change of behavior appears to be introduced by PR #810 - which more broadly introduced a method `heat_release_rate` that now masks the `extra` column of the `SolutionArray` object - the difference is the cylinder volume. A fix is pushed in #820.
